telicent_lib/exceptions.py

```diff
@@ -30,3 +30,13 @@ def __init__(self, component, config, message="A component is missing the config
         self.config = config
         self.message = message
         super().__init__(self.message)
+
+class KafkaTopicNotFoundException(Exception):
+    """
+    Raised when a specified topic (DataSource or DataSink) is not find on the specified server
+    """
+    def __init__(self, topic_name, message=None):
+        self.topic_name = topic_name
+        self.message = (message if message else f"Kafka topic {topic_name} not found on the specified bootstrap" +
```

Comment:
You should just be able to use a \ for the continuation, rather than combining 2 strings.